index.js

```diff
@@ -74,8 +82,12 @@ function getLocalNWVersion () {
     const nwManifest = await getLocalNwManifest();
     let localNwVersion = nwManifest?.version || '';
 
-    // '0.82.0-sdk' => '0.82.0'
-    localNwVersion = localNwVersion.replace('-sdk', '');
+    const parsedVersion = semver.parse(localNwVersion);
```

Comment:
Above this `localNwVersion` is set to `''`. I think `semver.parse('')` returns `null`, so you can't do `[null.major, null.minor, null.patch].join('.')`